.gitignore

```diff
@@ -0,0 +1,86 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# Distribution / packaging
```

Comment:
why do we need this file here?

Reply:
we must have .gitignore file in repo, kind of standard template keeping in mind the future development of the project